app/components/gh-distribution-action-select.hbs

```diff
@@ -0,0 +1,12 @@
+<span class="gh-select">
+    <OneWaySelect
+        @value={{this.distributionValue}}
+        @options={{this.availablePublishActions}}
+        @optionValuePath="value"
+        @optionLabelPath="name"
+        @includeBlank={{false}}
+        @update={{action "setPublishAction"}}
```

Comment:
`{{action}}` shouldn't be used any more unless you're dealing with an old non-Glimmer component and an actions hash. You've used the `@action` decorator so this can be `@update={{this.setPublishAction}}`.